src/components/ButtonBar.js

```diff
@@ -1,13 +1,27 @@
-
-
 const ButtonBar = () => {
-    const categories = ["All", "Mixes", "Music", "Comedy", "ReactJS", "Standup", "Cricket", "Mixes", "Mixes", "WebDev", "Travel", "Standup",]
+  const categories = [
+    "All",
+    "Mixes",
+    "Music",
+    "Comedy",
+    "ReactJS",
+    "Standup",
+    "Cricket",
+    "Mixes",
+    "Mixes",
+    "WebDev",
+    "Travel",
+  ];
   return (
-    <div>
-        {
-            categories.map((category) => <button key={category} className="py-1 px-4 m-3 bg-stone-100 rounded-lg">{category}</button>)
-        }
-      
+    <div className="sticky top-0 w-full bg-white z-10">
```

Comment:
just like top-0  we can use something [inset](https://tailwindcss.com/docs/top-right-bottom-left)

Reply:
Okayy